src/Sarif.Multitool.FunctionalTests/SarifMultitoolTestBase.cs

```diff
@@ -124,20 +124,14 @@ private static void SelectiveCompare(IDictionary<string, Rule> actualRules, IDic
             {
                 actualRules.Count.Should().Be(expectedRules.Count);
 
-                foreach (string key in actualRules.Keys)
+                for (int i = 0; i < actualRules.Count; ++i)
                 {
-                    Rule actualRule = actualRules[key];
-                    Rule expectedRule;
-                    expectedRules.TryGetValue(key, out expectedRule).Should().BeTrue();
+                    Rule actualRule = actualRules[i];
+                    Rule expectedRule = expectedRules[i];
```

Comment:
>[i] [](start = 53, length = 3)

Ah, I see. Assuming I wrote this code, it was all based on my misconception that you couldn't index into a list. Bleah.